### PR TITLE
Server statistics: New section for relays and nomads

### PR DESCRIPTION
**Pull request by annando**
There are so many relay servers out there that they deserve an own section. Also recently Mike referred to his projects as [Nomad Projects](https://socialhub.activitypub.rocks/t/hello-from-a-software-developer-and-social-adventurist/1829/7), so I think we can create a section for them as well (excluding Hubzilla, since they are big enough for their own section)

### CODE REVIEW

File: src/Module/Admin/Federation.php [hunk 1]
```diff
@@ -80,6 +82,10 @@ public static function content(array $parameters = [])
 
 				if (in_array($gserver['platform'], ['Red Matrix', 'redmatrix', 'red'])) {
 					$version['version'] = 'Red ' . $version['version'];
+				} elseif (in_array($gserver['platform'], ['osada', 'mistpark', 'roadhouse', 'zap'])) {
+					$version['version'] = $gserver['platform'] . ' ' . $version['version'];
+				} elseif (in_array($gserver['platform'], ['activityrelay', 'pub-relay', 'selective-relay', 'aoderelay'])) {
+						$version['version'] = $gserver['platform'] . '-' . $version['version'];
```
Comment (MrPetovan) [inline src/Module/Admin/Federation.php]: Wrong indentation.
```suggestion
					$version['version'] = $gserver['platform'] . '-' . $version['version'];
```

File: src/Module/Admin/Federation.php [hunk 2]
```diff
@@ -250,6 +268,68 @@ private static function reformaPleromaVersions(array $versionCounts)
...
+		return $versionCounts;
+	}
+
+	/**
+	 * Clean up relay version numbers
+	 *
+	 * @param array $versionCounts list of version numbers
+	 * @return array with cleaned version numbers
+	 */
+	private static function reformaRelayVersions(array $versionCounts)
```
Comment (MrPetovan) [inline src/Module/Admin/Federation.php]: Typo?
```suggestion
	private static function reformatRelayVersions(array $versionCounts)
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/Admin/Federation.php (hunk 2 above)
Comment (annando) [reply src/Module/Admin/Federation.php]: No, early morning coding.

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
Also added an unrelated change to this PR. In parallel I tested another iOS/Android app.

**Pull request merged**